examples/bunYoutubeScraper/bunYoutubeScraper.js

```diff
@@ -0,0 +1,54 @@
+// Get Write from Bun.
+const { write } = Bun;
+const BunYoutubeScraper = async (url, saveJson) => {
+  // GET the HTML of the page.
+  const body = await fetch(url);
+  // Parse the HTML.
+  const html = await body.text();
+  // Scrape the HTML.
+  const videoUrl = url;
+  const title = html.match(/title="(.*?)"/g)[2].split("\"")[1];
```

Comment:
Check out `HTMLRewriter` - it's a more durable way to implement this

`HTMLRewriter` is a global in Bun

Reply:
I've been practicing with "HTML rewriting tool", and it's perfect, the thing is that some elements can't be scraped with it. For example: the view counter is not found in an html tag, but in a json object.

Things that cannot be scraped with "HTML rewriting tool"
```
viewcounter
like counter
largeDescription
approxDurationMs
etc.
```
So for now the only method I can think of is to keep using the .match() method.